packages/fxa-payments-server/src/routes/Checkout/index.tsx

```diff
@@ -55,8 +132,138 @@ export const Checkout = ({
     return <PlanErrorDialog locationReload={locationReload} plans={plans} />;
   }
 
+  function refreshSubscriptions() {
+    // Will be added in https://mozilla-hub.atlassian.net/browse/FXA-3666
+    console.log('not yet implemented: refreshSubscriptions');
+  }
+
   return (
-    <div data-testid="checkout-route-container">{selectedPlan?.plan_id}</div>
+    <div data-testid="checkout-route-container">
+      <Header />
+      <div className="main-content">
+        <PaymentProcessing
+          provider="paypal"
+          className={classNames({
+            hidden: !transactionInProgress || paymentError,
+          })}
+        />
+        <SubscriptionTitle
+          screenType="create"
+          className={classNames({
+            hidden: transactionInProgress || paymentError,
+          })}
+        />
+        <div
+          className={classNames('product-payment', {
+            hidden: transactionInProgress || paymentError,
+          })}
+          data-testid="subscription-create"
+        >
+          <h2 className="step-header">1. Create a Firefox account</h2>
+          <hr />
+          {/* "Already have an account? <a>Sign In</a>" https://mozilla-hub.atlassian.net/browse/FXA-3664 */}
+          <NewUserEmailForm
+            setValidEmailSet={setValidEmailSet}
+            setAccountExists={setAccountExists}
+            checkAccountExists={checkAccountExists}
+          />
+
+          <hr />
+          <h2 className="step-header">2. Choose your payment method</h2>
+          <h4>required</h4>
```

Comment:
Is this heading markup just for the bold appearance/visual emphasis?  I don't think we should do that.